### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by tomByrer**
* should include "database" for SEO, since that is what people usually call them? ;)
* link to windows build steps
* link to blog; convienance & more SEO

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [![Circle CI](https://circleci.com/gh/compose/transporter.svg?style=svg)](https://circleci.com/gh/compose/transporter)
 
+Copmopse helps with database transportation and transformations from one store to another.
```
Comment (jipperinbham) [inline README.md]: small typo here, Copmopse -> Compose

also, transporter can sync from one to many stores but I'm fine leaving what you've put as is

**Comment by tomByrer** [comment]
Is that better?

**Comment by jipperinbham** [comment]
lgtm, thanks @tomByrer

**Comment by tomByrer** [comment]
Welcome!

**Pull request merged**